.github/workflows/rust.yml

```diff
@@ -32,41 +31,30 @@ jobs:
         toolchain: 1.53.0
         components: clippy, rustfmt
         default: true
```

Comment:
As these properties are set in this project these don't make any sense I think. 
so we set the installed toolchain as an override for the current dir with `default: true` and then specifies a toolchain?
not sure if this is correct or it is going to break later? 
![image](https://user-images.githubusercontent.com/55736468/133851000-41499b06-6938-4b79-8d1c-fe04960c1a14.png)

Not sure if we should just use stable and remove the default field?
so we will automatic follow along with rust stable. 
currently rust stable version is 1.55
`stable-x86_64-unknown-linux-gnu  - rustc 1.55.0 (c8dfcfe04 2021-09-06)`

Reply:
Following stable makes sense, let's do that.

The `default: true` does make sense in this case, although I think it's unnecessary. Basically you set a "default" toolchain (which in our case would be stable Rust), but then for each other step you can specify a toolchain to override it, e.g. Rust nightly.
However, setting the default only seems useful when you actually want to override it in some other step. It'll probably work without it.